cobalt/BUILD.gn

```diff
@@ -19,6 +19,8 @@ group("gn_all") {
   deps = [ "//starboard($starboard_toolchain)" ]
   if (!is_android) {
     deps += [ ":cobalt" ]
+  } else {
+    deps += [ "android:cobalt_apk" ]
```

Comment:
Can you make it so that it uses the existing subclasses in this folder for getting the correct user agent and storage paths?